docs/api/instruments/tti/thurlby1905a.rst

```diff
@@ -0,0 +1,24 @@
+############################################
+Thurlby 1905a intelligent digital multimeter
+############################################
+.. currentmodule:: pymeasure.instruments.tti.thurlby1905a
```

Comment:
We use `..autoclass` to import classes into the documentation, see any other instrument file.